dts/bindings/flash_controller/st,stm32-qspi-nor.yaml

```diff
@@ -0,0 +1,35 @@
+# Copyright (c) 2020, Linaro limited
+# SPDX-License-Identifier: Apache-2.0
+
+description: |
+    STM32 QSPI Flash controller supporting the JEDEC CFI interface
+
+    Representation of a serial flash on a quadspi bus:
+
+        mx25r6435f: qspi-nor-flash@0 {
+                compatible = "st,stm32-qspi-nor";
+                label = "MX25R6435F";
+                reg = <0>;
+                spi-max-frequency = <80000000>;
+                size = <0x4000000>;
+                status = "okay";
+        };
+
+compatible: "st,stm32-qspi-nor"
+
+include: ["flash-controller.yaml", "jedec,jesd216.yaml"]
+
+on-bus: qspi
+
+properties:
+    reg:
+      required: true
+    spi-max-frequency:
```

Comment:
This property name has to change, because [dtc infers](https://git.kernel.org/pub/scm/utils/dtc/dtc.git/tree/checks.c#n1090) bus type from the node name, or from property names in child nodes, so thinks this should be on a SPI bus, but it's not.  The Zephyr YAML `on-bus` stuff is not relevant to dtc.

Change it to `qspi-max-frequency` or `sck-frequency` or something that doesn't start with `spi`.

Reply:
Changed to `qspi-max-frequency`

Reply:
Wouldn‘t max-frequency be even better? It is in qspi node?